viewer/tsconfig.json

```diff
@@ -21,7 +21,12 @@
     "lib": [
         "dom",
         "esnext"
-    ]
+    ],
+
+    "baseUrl": "./src/",
+    "paths": {
+      "@/*": ["*"]
```

Comment:
Finally the file I was looking for :) An opinion here and maybe I am wrong. @ alone seems so weird. I would prefer if it was say:
  "@datamodels/*": ["datamodels/*"],
  "@utilities/*": ["utilities/*"]
And honestly, the alias feels even more useful if used to shorten imports so:
  "@cad/*": ["datamodels/cad/*"],
  "@pointcloud/*": ["datamodels/pointcloud"]

Thoughts?

Reply:
See answer above